### PR TITLE
Solution for HW_1.1 

### PR DESCRIPTION
**Pull request by lord-dwarf**

### CODE REVIEW

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 1]
```diff
@@ -15,6 +15,12 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        return (result1 == bet1) && (result2 == bet2)
+                ? 2
+                : ((result1 > result2) && (bet1 > bet2)) || ((result1 < result2) && (bet1 < bet2))
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FootballMatch.java]:
```suggestion
                ? 2 : ((result1 > result2) && (bet1 > bet2)) || ((result1 < result2) && (bet1 < bet2))
```

Comment (dmytro-hryhoruk) [reply src/main/java/core/basesyntax/FootballMatch.java]: same i other lines

**Comment by dmytro-hryhoruk** [comment]
too many '?' conditions, try to simplify it

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 2]
```diff
@@ -15,6 +15,8 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        return (result1 == bet1) && (result2 == bet2) ? 2 : (result1 > result2) && (bet1 > bet2)
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FootballMatch.java]: Almost there) '?' should be on a new line

File: src/main/java/core/basesyntax/FootballMatch.java (hunk 1 above)
Comment (lord-dwarf) [reply src/main/java/core/basesyntax/FootballMatch.java]: done

**Review by dmytro-hryhoruk — APPROVED** [review]

File: src/main/java/core/basesyntax/FootballMatch.java (hunk 2 above)
Comment (lord-dwarf) [reply src/main/java/core/basesyntax/FootballMatch.java]: done

**Pull request closed without merging**